skglm/tests/test_estimators.py

```diff
@@ -164,6 +168,89 @@ def test_mtl_path():
     np.testing.assert_allclose(coef_ours, coef_sk, rtol=1e-5)
 
 
+def test_CoxEstimator():
+    try:
+        from lifelines import CoxPHFitter
+    except ModuleNotFoundError:
+        pytest.xfail(
+            "Testing Cox Estimator requires `lifelines` packages\n"
+            "Run `pip install lifelines`"
+        )
+
+    reg = 1e-2
+    n_samples, n_features = 100, 30
+    random_state = 1265
```

Comment:
no need to declare variable if used only once IMO

Reply:
Agree!
Though, if you don't have a strong opinion on it, I would rather stick with it as it makes debugging easier.